templates/daily_post.j2

```diff
@@ -20,6 +20,12 @@ Welcome to the [Seattle Reddit Community](https://www.reddit.com/r/SeattleWA) Da
 
 ---
 
+Quote of the Day:
```

Comment:
So here it would be (assuming we throw in the weekday) something like:

```
---

{% if day_of_week == 4 %}
Haiku Friday:
{% else %}
Quote of the Day:
{% endif %}

{{ qotd }}
```

Reply:
Great idea, that hadn't even occured to me! I'll go ahead and build that in.